src/dispatch.rs

```diff
@@ -52,27 +52,62 @@ pub struct Dispatcher<'c, 't, C = ()> {
 impl<'c, 't, C> Dispatcher<'c, 't, C>
     where C: Clone + Send + 'c
 {
-    /// Dispatches the systems given the
+    /// Dispatch systems with given resources and context.
+    ///
+    /// This function automatically redirects to
+    ///
+    /// * [`dispatch_par`] in case the `parallel` feature is set
+    /// * [`dispatch_seq`] in case the `parallel` feature is not set
+    ///
+    /// [`dispatch_par`]: struct.Dispatcher.html#method.dispatch_par
+    /// [`dispatch_seq`]: struct.Dispatcher.html#method.dispatch_seq
+    pub fn dispatch(&mut self, res: &mut Resources, context: C) {
+        self.dispatch_cfg(res, context);
```

Comment:
Why do you need this helper function? You can use the configuration flags directly here, can't you?